rebar.config

```diff
@@ -1,13 +1,18 @@
 {erl_opts, [debug_info]}.
 
-{deps, [{epgsql, "3.4.0"},
```

Comment:
I would probably keep an empty array of deps here, it should be clear that there are regular deps and then dev deps